### PR TITLE
Swift Package Manager (SPM) Support

### PR DESCRIPTION
**Pull request by amrangry**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SVProgressHUD",
+    platforms: [.iOS(.v10)],
```
Comment (martin-key) [inline Package.swift:8]: Isn't that working on macOS as well?

**Comment by callo90** [comment]
Any updates?

**Comment by OliverLetterer** [comment]
⬆️ +1

**Comment by hakanyildizay** [comment]
Do you have plans about this PR?

**Comment by sumofighter666** [comment]
Close this PR. It doesn't work.

**Comment by Kaspik** [comment]
It needs some changes made in https://github.com/SVProgressHUD/SVProgressHUD/pull/1063, this PR doesn't actually PR.

**Comment by honkmaster** [comment]
I've updated the component and addressed several bugs. Please review version 2.3.0, and should you find any remaining issues, your feedback would be greatly valued.

**Pull request closed without merging**